<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeapPleaLongDialog</class>
 <widget class="QDialog" name="LeapPleaLongDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2500</width>
    <height>3500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>49</green>
        <blue>91</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Leap Plea Case Information</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 49, 91);
font: 75 10pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(25, 49, 91);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 127);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>941</width>
           <height>779</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" rowstretch="1,1,2,1">
          <item row="3" column="0">
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Time to Complete LEAP:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="time_to_complete_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>120 days</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>forthwith</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Sentencing Date:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="sentencing_date">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="print_entry_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Print and Open Entry</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="create_entry_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 255, 0);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Open Entry</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="close_dialog_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 58, 63);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Close Dialog</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QFrame" name="frame_6">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0" colspan="2">
              <layout class="QGridLayout" name="charges_gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="offense_label_1">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Offense:</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Dismissed:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_21">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Degree:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="plea_label_1">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Plea:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_20">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Statute:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="add_charge_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(62, 146, 255);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Add Charge</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="guilty_all_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Guilty All</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="Line" name="line">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Palatino Linotype'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;INSTRUCTIONS: &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;Must have completed LEAP eligibility checklist and LEAP Participation agreement that is signed by Defendant, LEAP Coordinator and Prosecutor. Defendant must plead guilty to all charges. May enter program even if Prosecutor doesn't agree to dismiss companion charges, but Defendant must understand and be made aware that companion charges will not be dismissed and that he will be sentenced separately on companion charge(s) at completion of LEAP Program.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QFrame" name="case_name_Frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="3">
              <widget class="QCheckBox" name="defense_counsel_waived_checkBox">
               <property name="text">
                <string>Defendant waived right to counsel</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QComboBox" name="defense_counsel_type_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>Public Defender</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Private Counsel</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Def. Counsel:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="case_number_lineEdit">
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Case Number:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QPushButton" name="clear_fields_case_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Clear Fields</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Def. Last Name:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="defendant_last_name_lineEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="cancel_Button">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(160, 160, 160);</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QDateEdit" name="plea_trial_date">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="minimumDate">
                <date>
                 <year>2021</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="date">
                <date>
                 <year>2021</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="defense_counsel_name_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="defendant_first_name_lineEdit">
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Def. First Name:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Appearance Reason:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="appearance_reason_box">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>arraignment</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>change of plea</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>trial to court</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>jury trial</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>LEAP sentencing</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defendant_first_name_lineEdit</tabstop>
  <tabstop>defendant_last_name_lineEdit</tabstop>
  <tabstop>case_number_lineEdit</tabstop>
  <tabstop>defense_counsel_type_box</tabstop>
  <tabstop>defense_counsel_waived_checkBox</tabstop>
  <tabstop>time_to_complete_box</tabstop>
  <tabstop>sentencing_date</tabstop>
  <tabstop>create_entry_Button</tabstop>
  <tabstop>print_entry_Button</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>close_dialog_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>create_entry()</slot>
  <slot>continue_dialog()</slot>
  <slot>close_window()</slot>
  <slot>set_dialog()</slot>
  <slot>update_case_information()</slot>
  <slot>ovi_dialog()</slot>
  <slot>sentencing_dialog()</slot>
  <slot>proceed_to_sentencing()</slot>
  <slot>proceed_to_ovi()</slot>
  <slot>add_charge()</slot>
  <slot>set_statute()</slot>
  <slot>set_pay_date()</slot>
  <slot>amend_offense()</slot>
  <slot>set_offense()</slot>
  <slot>add_charge_to_view()</slot>
  <slot>delete_charge()</slot>
  <slot>delete_charge_from_view()</slot>
  <slot>close_event()</slot>
  <slot>set_fra_in_file()</slot>
  <slot>set_fra_in_court()</slot>
  <slot>start_add_conditions_dialog()</slot>
  <slot>start_amend_offense_dialog()</slot>
 </slots>
</ui>
